conjur/logic/login_logic.py

```diff
@@ -41,10 +43,13 @@ def get_api_key(cls, ssl_verify, credential_data, password, conjurrc):
         elif ssl_verify and credential_data.machine.startswith("https"):
             # Catches the case where a user does not run in insecure mode but the
             # .conjurrc cert_file entry is empty
-            if conjurrc.cert_file == '':
-                certificate_path = True
-            else:
-                certificate_path = conjurrc.cert_file
+            if conjurrc.cert_file == '' and ssl_verify:
```

Comment:
how is this section related to support of CA signed certificates ?

Reply:
if upon initialization of the CLI, the user runs `conjur --insecure init -u https://conjur-server` then
 ssl_verify=False and the cert_file param in the conjurrc will be set to ''

```
---
account: cucumber
appliance_url: https://conjur-server.pem
cert_file: ''
plugins: []
```
So this check is to determine if the cli was initialized in insecure mode but the follow-up command (login) was run without `--insecure`